anaconda.spec.in

```diff
@@ -381,6 +386,7 @@ rm -rf \
   %{buildroot}/%{_sysconfdir}/xdg/autostart/liveinst-setup.desktop \
   %{buildroot}/%{_bindir}/liveinst \
   %{buildroot}/%{_libexecdir}/liveinst-setup.sh \
+  %{buildroot}/%{_libexecdir}/locale1-x11-sync \
```

Comment:
I see, you need to add also service file here or we will break ELN builds.

Reply:
Oh right, there is a special section for this - not very intuitive. :P Anyway, should be added now. :)